README.md

```diff
@@ -20,7 +20,12 @@ It's a pretty standard Rails app, clone it, `bundle` it.
 $ PLEK_SERVICE_STATIC_URI=assets.digital.cabinet-office.gov.uk ./startup.sh
 ```
 
-The application will start on port `3133`, so to access it visit [0.0.0.0:3113](http://0.0.0.0:3113/)
+If running on a VM, then Rails needs to be bound to `0.0.0.0` rather than localhost so that it can be accessed outside of the VM:
+```
+PLEK_SERVICE_STATIC_URI=assets.digital.cabinet-office.gov.uk bundle exec rails s -b 0.0.0.0 -p 3113
```

Comment:
I don't think the hosting binding param is needed outside a GOV.UK VM, only for the VM case below.